tests/data_context/store/test_store_backends.py

```diff
@@ -1325,6 +1325,28 @@ def test_InlineStoreBackend(empty_data_context) -> None:
     inline_store_backend.remove_key(tuple_)
 
 
+def _create_datasource_config(regex_pattern: str = "(.+)\\.csv") -> dict:
+    datasource_config_string = f"""
+        class_name: Datasource
+
+        execution_engine:
+            class_name: PandasExecutionEngine
+
+        data_connectors:
+            my_other_data_connector:
+                class_name: ConfiguredAssetFilesystemDataConnector
+                base_directory: my/base/dir/
+                glob_directive: "*.csv"
+
+                default_regex:
+                    pattern: {regex_pattern}
+                    group_names:
+                        - name
+        """
+    datasource_config: dict = yaml.load(datasource_config_string)
+    return datasource_config
```

Comment:
This is a legacy config do we want to support it?

Reply:
Hmm so I just reused the existing test data because it was there and easy to reuse. You're saying just update test code to use FDS, or is there something more that I'm missing around the backend itself? (Totally happy to update test data fwiw)

Reply:
I'd say let's use FDS if we can since we going to rip out legacy datasources before we get rid of `InlineStoreBackend`

Reply:
Updated!